docs/genindex.html

```diff
@@ -106,14 +108,20 @@ <h2 id="C">C</h2>
 </li>
       <li><a href="index.html#plaid.Client">Client (class in plaid)</a>
 </li>
-  </ul></td>
```

Comment:
I simply ran `make docs`. Things not related to payment initiation got added here because `make docs` wasn't run on earlier python client library changes (as it should have been).

I viewed the generated documentation on my local browser and it looks fine.